packages/shared_preferences/shared_preferences_platform_interface/lib/shared_preferences_platform_interface.dart

```diff
@@ -47,7 +47,7 @@ abstract class SharedPreferencesStorePlatform {
   bool get isMock => false;
 
   /// Removes the value associated with the [key].
-  Future<bool> remove(String key);
+  Future<bool?> remove(String key);
```

Comment:
When does `remove` return `null`, and what does it mean in terms of a consumer of this API?